README.md

```diff
@@ -1,5 +1,9 @@
 <p align="center"><a href="https://usememos.com"><img height="64px" src="https://raw.githubusercontent.com/usememos/memos/main/resources/logo-full.webp" alt="✍️ memos" /></a></p>
 
+<p align="center">
+  <b>A customizable, self-hosted alternative to Twitter</b>
```

Comment:
At this stage, we may not have this capacity. 

But it's our main roadmap. Let's introducing this later.